src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -17,7 +17,7 @@ import HoverTooltip from "./HoverTooltip";
 
 type CanvasHoverTooltipProps = {
   dataset: Dataset | null;
-  featureKey: string;
+  featureKey: string | null;
```

Comment:
code smell: why did this get changed to allow null everywhere?

Reply:
The selected feature key is invalid whenever there isn't a dataset loaded. Before, we were representing this invalid state with an empty string (`""`). 

It's possible for a feature key to be keyed with the empty string-- it's not behavior I'd want scientists to use, but it feels more obvious to explicitly type `featureKey` as `null` whenever it's invalid.